default.config.js

```diff
@@ -61,6 +61,18 @@ module.exports = {
       pipeWarnTime: 500,
       pipeTimeout: 5000,
       initTimeout: 10000,
+    },
+    codes: {
```

Comment:
* wording: `codes` sounds too generic to me.. `errorCodes` or maybe `errorDomains` ?
* Please use the dedicated plugin entry (i.e. `plugins` > `kuzzle-plugin-logger` > `errorCode`)

Reply:
This codes is like an id per plugin (this is the `subdomain`), they are not error codes, maybe it should be named `id` or `pluginCode` ?

Reply:
Hum... you're right, calling it an error code is not good either. 
I cannot think of better alternative than `code` after all, even if I still don't like it.

The remark about the plugin entry still applies though.